TTS/speaker_encoder/utils/prepare_voxceleb.py

```diff
@@ -27,10 +27,8 @@
 
 import pandas
```

Comment:
pandas is missing in the requirements

Reply:
Hmm, looks like it was missing before as well. I added it without any version pinning since I can't test it either. Something you might want to take a look at after this PR.